locales/en/dashboard.ftl

```diff
@@ -0,0 +1,134 @@
+# Variables:
+#   $nr (number) - Number of a particular type of exposure found for the user
+exposure-chart-legend-value-nr = { $nr }×
+exposure-chart-caption = This chart shows how many times your info is actively exposed.
+# Variables:
+#   $total_fixed_exposures_num (number) - Number of fixed exposures
+#   $total_exposures_num (number) - Number of total exposures
+exposure-chart-caption-fixed = This chart shows the total exposures that are fixed ({ $total_fixed_exposures_num } out of { $total_exposures_num })
+exposure-chart-returning-user-upgrade-prompt = Home address, family members and more are not yet included.
+exposure-chart-returning-user-upgrade-prompt-cta = Start a free scan
+exposure-chart-scan-in-progress-prompt = <b>Scan in progress:</b> address, family members, and more are not yet included.
+
+modal-active-number-of-exposures-title = About your number of active exposures
+# Variables:
+#   $limit (number) - Number of email addresses included in the plan
+modal-active-number-of-exposures-part-one-all =
+  { $limit ->
+    [one] This chart includes the total number of times we found each type of data exposed across all data breaches for the { $limit } email address that you are currently monitoring.
+    *[other] This chart includes the total number of times we found each type of data exposed across all data breaches for up to { $limit } email addresses that you are currently monitoring.
+  }
+modal-active-number-of-exposures-part-two = For example, if you have 10 exposures of your phone number, that might mean one phone number is exposed across 10 different sites, or it could mean 2 different phone numbers were exposed across 5 different sites.
+modal-active-number-of-exposures-part-three = This chart does not include any exposures that are in-progress of being auto-removed. Once your exposures are fixed, they will be added to your total number of fixed exposures on the Fixed page.
+
+modal-cta-ok = OK
+modal-open-alt = Open
+modal-close-alt = Close
+
+progress-card-heres-what-we-fixed-headline = Here’s what we fixed
+progress-card-manually-fixed-headline = Manually fixed
+
+dashboard-tab-label-action-needed = Action needed
+dashboard-tab-label-fixed = Fixed
+# Variables:
+# $exposures_unresolved_num is the unresolved number of exposures the user has.
+# $data_breach_unresolved_num is the unresolved number of data breaches the user has.
+# $data_broker_unresolved_num is the unresolved number of data brokers selling the user’s data.
+dashboard-exposures-area-description = We found your information exposed { $exposures_unresolved_num } times over { $data_breach_unresolved_num } data breaches and { $data_broker_unresolved_num } data broker sites that are selling your personal info.
+dashboard-exposures-all-fixed-label = All fixed here!
+
+
+dashboard-exposures-area-headline = View all sites where your info is exposed
+dashboard-fixed-area-headline-all = View all exposures that are fixed
+
+dashboard-exposures-filter = Filter
```

Comment:
Is this an action (e.g. button) or a title?

Reply:
eb582afb5